bin/reference_tools/update_bed.pl

```diff
@@ -6,8 +6,26 @@
 use vcf2 qw( parse_vcf );
```

Comment:
This changes are from the previous initial PR. It would have been better to make a new PR. This does not need to be reviewed.